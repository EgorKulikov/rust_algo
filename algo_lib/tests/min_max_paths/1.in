3
5 3
1 2
2 3
3 4
4 5
2 2
1 2
4 1
1 2
1 3
2 4
