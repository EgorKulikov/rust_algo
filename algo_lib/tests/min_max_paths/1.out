3 5 0 7 8
2 0
0 2 3 4
